src/main/java/io/jenkins/security/csp/Scanner.java

```diff
@@ -39,7 +40,8 @@ public class Scanner {
 
     protected static final Map<String, Pattern> JAVA_PATTERNS = Map.of("Inline Event Handler (Java)", Pattern.compile("(?<![a-z0-9])" + JS_EVENT_ATTRIBUTES + "=.*?((?<!\\\\)\")", Pattern.CASE_INSENSITIVE),
             "Inline Script Block (Java)", Pattern.compile("(<script>|<script(|\\s[^>]*)[^/]>)\\s*?(?!</script>)\\S.*?</script>", Pattern.DOTALL | Pattern.CASE_INSENSITIVE),
-            "FormApply#applyResponse", Pattern.compile("FormApply[.]applyResponse[(].*"));
+            "FormApply#applyResponse", Pattern.compile("FormApply[.]applyResponse[(].*"),
+            "Javascript scheme (Java)", Pattern.compile("\".*(?<![a-z0-9])javascript:.*?((?<!\\\\)\")", Pattern.CASE_INSENSITIVE));
```

Comment:
This starts with a greedy `.*` following a `"` and I expect giant matches in any nontrivial Java files. Did you test this with any plugins you can find in [GH code search](https://github.com/search?q=org%3Ajenkinsci%20%22javascript%3A%22%20path%3A**%2F*.java&type=code) to see how this behaves?

Reply:
>This starts with a greedy .* following a " and I expect giant matches in any nontrivial Java files.
Well I'm not using `Pattern.DOTALL`, so it's not going to be a multiline catch.

I wanted to have all the String using the scheme to have the full context in the report.

>Did you test this with any plugins you can find in GH code search to see how this behaves?

Yes, I tried it and got the results I expected. Feel free to adapt it as needed.